bokehjs/test/index.coffee

```diff
@@ -30,3 +30,7 @@ blacklist.push('constructor')
 for own key, val of global.window
   if blacklist.indexOf(key) == -1
     global[key] = val
+
+global.HTMLElement = global.window.HTMLElement
+global.Event = global.window.Event
+global.Image = global.window.Image
```

Comment:
Apparently we were doing this wrong all the time. Those new lines are just to make node/jsdom happy, because `HTMLElement` at al. are special and they don't enumerate properly (as I would expect, but no worries, similar issue arises in web browsers). We should follow https://github.com/tmpvar/jsdom/wiki/Don't-stuff-jsdom-globals-onto-the-Node-global. Unfortunately this is incompatible with how we use mocha. That wiki expects us to organize our code as `mocha -> jsdom`, but we need `jsdom -> mocha`. Perhaps we should get rid of `gulp-mocha` and run mocha manually from within a jsdom env. I don't know. Tests work in this PR, so may proceed with this hack.